packages/addons/addon-depends/rpi-tools-depends/lan951x-led-ctl/package.mk

```diff
@@ -0,0 +1,41 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2016 Team LibreELEC
```

Comment:
@lrusak nit-pick but it should be `2017-present` (if the package is created 2017)
same for the backport